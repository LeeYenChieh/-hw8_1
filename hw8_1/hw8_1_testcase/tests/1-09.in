1WLuJnC67tFACUbHFG5OKRoc6V5PdUP9 vsMpsIfhNb0Tz38QDIQH
knKeGDdOLi8V5wf5neBVaq7vM3T shRsAd0R
Zv7urh4pDTO3X1SALWBVyLzK3 Y6xNXUOdEI1q6kJaCV0AM5ohwEqEQ08Ik0WT8vIzl5r
ysYt N6YaFPSt7AfSgkxlEuhnIlMdyHQODuOFj
f67A5WOEjP8xXO5mXdGkUqHk2gsiCHAc2p1WmBA3scj3yP b2e4AXOb6OnaFCqvtlBhZMJBBu2ieUL3Onezlkn5OX9QPfIKwl5vJRY5U
1WLuJnC67tFACUbHFG5OKRoc6V5PdUP9 display
knKeGDdOLi8V5wf5neBVaq7vM3T display
Zv7urh4pDTO3X1SALWBVyLzK3 display
1WLuJnC67tFACUbHFG5OKRoc6V5PdUP9 pa0ZikXUpZwuyQbSdEyuUXXI
ilyAFnFW85ufrRInX4uYsIIeQwlsN Cod0C6LssXcaoIowQ6UYkBEbkbJuKTQ1cxBrR4Iy
