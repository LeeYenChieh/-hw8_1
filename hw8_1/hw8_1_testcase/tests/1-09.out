vsMpsIfhNb0Tz38QDIQH 
shRsAd0R 
Y6xNXUOdEI1q6kJaCV0AM5ohwEqEQ08Ik0WT8vIzl5r 
